### ISSUE TITLE
[11.0] Base Upgrade does not work after incomplete view validation fix

### ISSUE THREAD

**Issue by wtaferner** [odoo/odoo#21561]
Boolean is not a field ;-)

https://github.com/odoo/odoo/commit/7b3796d77c198553b1f9fe422a46eb477df3e3f3#commitcomment-26175516

@rco-odoo @odony @nim-odoo 
Not sure if you guys got my ping on the commit itself, but as it is stable and global, please take care of this for me ASAP. Thank you in advance.

OPW-787427

**Comment by odony** [comment]
Sure we got the ping, @rco-odoo is on it, thanks for notifying us thoroughly ;-)

**Comment by wtaferner** [comment]
@rco-odoo 
As it does also affect every upgrade of any module can you please revert the poisoned commit in case you need more time to fix it correctly? At the moment the stable branch is unusable with the commit.

Thank you and I hope you have the correct solution soon.

@odony 
Well, as soon you give me feedback that you are working on it, I am perfectly fine and stop spamming and notifying over all channels I have in order to get through to the people who can actually change and improve things ;-)

**Comment by yvaucher** [comment]
Here the error, as @wtaferner said would be better to revert that commit.

```
    raise ValidationError("%s\n\n%s" % (_("Error while validating constraint"), tools.ustr(e)))
ParseError: "Error while validating constraint
Field 'False' used in attributes must be present in view but is missing:
 - 'False' in domain="['|', ('active', '=', False), ('active', '=', True)]"
Field 'True' used in attributes must be present in view but is missing:
 - 'True' in domain="['|', ('active', '=', False), ('active', '=', True)]"
Error context:
View `ir.cron search`
[view_id: 25, xml_id: n/a, model: ir.cron, parent_id: n/a]
```

**Comment by Yenthe666** [comment]
Yep a revert would be nice, one of the guys at my company was bitten by this too.

**Comment by wtaferner** [comment]
@odony 
Thank you for fixing it even though it was "only" a Python 2 issue. It was not clear for me as I was bumping into it directly in my dev environment which is currently still on Python 2 and later tested before deployment in Python 3. So, no need to plan any upgrade :-)
I hope you understand that I still reported it with high prio as it was not just a minor showstopper even if it is unsupported ;-)

**Comment by Yenthe666** [comment]
@wtaferner always the test environments that bite you with these things hmm. 
Thanks for the fix @odony
